packages/shared/src/components/plus/PlusCheckoutContainer.tsx

```diff
@@ -40,16 +41,26 @@ export const PlusCheckoutContainer = ({
     return null;
   }, [isPlusAvailable, giftToUser, isPlus]);
   const shouldRenderCheckout = !ContainerElement;
+  const showTrialCheckbox =
+    isFreeTrialExperiment &&
+    !giftToUser &&
+    !isPricesPending &&
+    shouldRenderCheckout;
 
   return (
-    <div
-      ref={shouldRenderCheckout ? checkoutRef : undefined}
-      className={classNames(
-        shouldRenderCheckout && 'checkout-container',
-        className?.container,
-      )}
-    >
+    <div className={className?.container}>
+      <div
+        ref={shouldRenderCheckout ? checkoutRef : undefined}
+        className={classNames(shouldRenderCheckout && 'checkout-container')}
+      />
       {ContainerElement && <ContainerElement className={className?.element} />}
+      {showTrialCheckbox && (
+        <div className="mx-auto mt-4 max-w-[40rem]">
```

Comment:
Isn't that just this:
```suggestion
        <div className="mx-auto mt-4 max-w-10">
```

Reply:
Not really, I need 40 * 1rem size (~640px)

Reply:
Ah I thought it was tailwind class but it's 42 I see.
We have a lot of 40rem overrides, probably at some stage start thinking of adding them as values.